Products/CMFPlone/browser/templates/plone-overview.pt

```diff
@@ -31,14 +31,9 @@
                 i18n:attributes="alt" /></p>
         <h1 i18n:translate="">Plone is up and running.</h1>
         <p class="lead">
-            <span i18n:translate="label_plone_com_description"> For an introduction to Plone, success stories, demos, providers, visit</span>
-            <a href="http://plone.com"
-              title="Plone.com"
-              target="_blank"
-              i18n:attributes="title label_plone_com_title;">plone.com</a>.
-            <span i18n:translate="label_plone_org_description"> For documentation, add-ons, support, community, visit</span>
+            <span i18n:translate="label_plone_org_description"> For an introduction to Plone, documentation, demos, add-ons, support, and community, visit</span>
```

Comment:
I think it is appropriate to omit "success stories" as the reader is already sold on installing Plone and trying it.

Reply:
But `success stories` option is already omitted! Sir...

Reply:
Yes, I was merely stating why I think it should be omitted, in case someone else notices it and wants it reinstated just for the sake of reinstatement. It should only be put back if it is reasonable to do so.

Reply:
Okay! hahah😅
So shall I marks this as resolved? Or keep it like this only?

Reply:
It is OK to resolve. It can be unresolved, if needed, until it is merged.

Reply:
Agree with replacing 'success stories' with 'documentation'

Reply:
Seems good.